dashboard/client/src/components/LogView/LogVirtualView.tsx

```diff
@@ -206,7 +208,7 @@ const LogVirtualView: React.FC<LogVirtualViewProps> = ({
         ...style,
       }}
       itemSize={fontSize + 6}
-      itemCount={total}
+      itemCount={total > maxLines ? maxLines : total}
```

Comment:
No, we need to limit the amount of data we fetch from the API, not just limit the number of lines we render

Reply:
Remember, i showed you that all the log fetching endpoints accept a `lines=x` parameter to limit the number of lines.

Reply:
Sounds great, thanks